tests/data/basic.sssom.rdf

```diff
@@ -142,7 +142,7 @@ x:bone_element owl:equivalentClass y:bone,
             sssom:confidence "0.8407144059077161"^^<xsd:double> ;
             sssom:mapping_tool "rdf_matcher" ;
             sssom:match_string "appendag" ;
-            sssom:match_type "Lexical|Unspecified" ;
+            sssom:mapping_justification "semapv:UnspecifiedMatching" ;
```

Comment:
this should be:

```
sssom:mapping_justification semapv:UnspecifiedMatching
```

not

```
sssom:mapping_justification "semapv:UnspecifiedMatching"
```

(quotes). See https://github.com/mapping-commons/sssom/blob/master/src/sssom_schema/schema/sssom_schema.yaml#L213

Reply:
This issue here still needs to be resolved.